pkgs/development/python-modules/urllib3/default.nix

```diff
@@ -4,6 +4,8 @@
 
 buildPythonPackage rec {
   pname = "urllib3";
+  # XXX: snowflake-connector-python v1.8.5 depends on urllib3 1.21.1 >= version <1.25
```

Comment:
This can be removed. Nearly the whole of python-packages.nix depends on this package through botocore/boto.

Reply:
done.